apps/web/pages/404.tsx

```diff
@@ -73,7 +80,7 @@ export default function Custom404() {
                 <ul role="list" className="mt-4">
                   <li className="border-2 border-green-500 px-4 py-2">
                     <a
-                      href="https://console.cal.com"
+                      href={CONSOLE_URL}
                       className="relative flex items-start space-x-4 py-6 rtl:space-x-reverse">
```

Comment:
this always has to stay console.cal.com, no ned to add CONSOLE_URL to consts

Reply:
@PeerRich but why is the variable available?